packages/fxa-payments-server/src/components/PayPalButton/index.tsx

```diff
@@ -216,9 +214,9 @@ export const PaypalButton = ({
   return (
     <>
       <div
-        className={classNames({
-          'disabled-overlay': disabled,
-        })}
+        className={disabled
+          ? "relative after:absolute after:bg-white after:content-[''] after:opacity-60 after:top-0 after:left-0 after:w-full after:h-full after:z-[1000]"
```

Comment:
Lowering the z-index did not seem to work.

Reply:
Just a note to add here: looks like z-index style(s) in the iframe is(are) set really high. I only went one level deep, but found it was set to `z-index: 200`. I don't think it needs to be as high as 1000 but I am totally cool with keeping it this high as it's not affecting the other components, and it is a good fail-safe in case PayPal updates their code and, for whatever reason, goes higher.

Reply:
Ah, this makes sense - I tried `z-50` per Tailwind documentation, and it did not work. For now, I'm also aligned with keep this at 1000 and perhaps re-visiting this. :)